tgui/packages/tgui/components/Input.tsx

```diff
@@ -104,6 +108,7 @@ export const Input = (props: Props) => {
       <div className="Input__baseline">.</div>
       <input
         className="Input__input"
+        disabled={disabled}
```

Comment:
Not in the cl: we can now disable inputs, some components were using this to no effect before